### PR TITLE
F/setup

### PR DESCRIPTION
**Pull request by lRebornsl**
# 🎏   In this *Milestone*  🙋‍♂️⁉:
- ✅ Setup project with create-react-app.
- ✅ No linters error.
- ✅ Follow the best GitHub flow.

## ⭐To the code reviewer 👨‍💻 
- I really thank 🙇‍♂️  you, code reviewer 👨‍🎨  for dedicating your precious 🥇  time ⌚  to check the PR 🧾 .
- I kindly ask 🙏 code reviewer 🤵 that if there is any issue 🦯 in this project, please do list 📃 them in a descriptive 💡 manner and give your best suggestions 🎁 if needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+- **Do math operations.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming soon](#)
```
Comment (addod19) [inline README.md]: - [x] It would be better if we remove the live demo link since it is not working as expected, the link is misleading

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @lRebornsl, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] App runs without errors :100: 
- [ ] Script works as expected :rocket: 
- [ ] App created with `create-react-app` :1st_place_medal: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (lRebornsl) [reply README.md]: solve

**Review by Reem-lab — APPROVED** [review]
# Hi @lRebornsl,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ] No Linters errors 🔥
- [ ]  Great job working on the setup💯
- [ ]  Followed Gitflow 🔥
- [ ]  Kudos for making the requested changes 🤩


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**